tests/drivers/i2s/i2s_speed/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 cmake_minimum_required(VERSION 3.13.1)
+
+if(BOARD STREQUAL "96b_argonkey")
```

Comment:
But 96b_argonkey.conf is only needed when using this DMA driver, and also it's content depends on our application. In this test 1024 is enough since only DMA driver requires heap memory, while it may be not sufficient for other applications.

Reply:
You're right about this, I misunderstood what you said.